### PR TITLE
build: remove obsolete config from pom.xml

### PR DESCRIPTION
**Pull request by padamstx**
## PR summary
Removes some obsolete configuration from the pom.xml file.

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [ ] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [ ] No

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -410,7 +405,6 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <configuration>
-                    <aggregate>true</aggregate>
```
Comment (pyrooka) [inline pom.xml:413]: I don't know what this line is for, but if you removed it on purpose, you get my approval! 😄

**Review by pyrooka — APPROVED** [review]
LGTM!

File: pom.xml (hunk 1 above)
Comment (padamstx) [reply pom.xml:413]: yes, it is apparently no longer a valid config property for the javadoc plugin as it was giving us a warning.

**Pull request merged**